### DISCUSSION TITLE
Usage statistics?

### DISCUSSION THREAD

**Discussion by Spsrky** [code4recovery/12-step-meeting-list#860] [Q&A] [answered]
I report website traffic at our monthly Intergroup meeting, including how much traffic the Meeting List page has gotten. Is there any way to measure how many hits we've had through the smartphone app or maybe how many user phones have accessed the JSON file, if that makes sense? Thanks!

Theron

**Comment by tim-rohrer** [comment, marked as answer]
TSML doesn't track any usage statistics. 

But, tracking page views is something you can do with other Wordpress plugins. Companies like Automattic might be of use to you. I've not worked extensively with any of these so you're better off just doing some searching and experimenting.
